engine/runtime/examples/plateau/testdata/workflow/example_bulk_renamer/all_add_prefix.yml

```diff
@@ -0,0 +1,53 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/reearth/reearth-flow/main/engine/schema/workflow.json
+id: 00caad2a-9f7d-4189-b479-153fa9ea36dc
```

Comment:
@soma00333 
I'll try to make it easier to write test cases. We will try to send you an example later.

Reply:
@soma00333 
I have moved it from examples to tests.
I have written three tests for now, but I would appreciate it if you could also prepare test results (expect.json) for other test cases.

https://github.com/reearth/reearth-flow/pull/583/commits/b7ba4ce1e2791444ea1ccce4be1f1648dff1ecef

Reply:
@miseyu 
Thank you. I’ll take care of the remaining tests👍

Reply:
Fixed here https://github.com/reearth/reearth-flow/pull/583/commits/db4e709104ded3ca17bfc56bb7d11c4d3beb636e